support/murmurhash/src/main/scala/sjsonnew/support/murmurhash/Converter.scala

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright (C) 2016 Eugene Yokota
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package sjsonnew
+package support.murmurhash
+
+import scala.util.Try
+import HashUtil.hashLong
+import java.lang.{ Double => JDouble }
+import scala.util.hashing.MurmurHash3
+
+object Converter extends SupportConverter[Int] {
+  private val nullHash = 0xc0
+  private val falseHash = 0xc2
+  private val trueHash = 0xc3
+
+  def hash[A](obj: A)(implicit writer: HashWriter[A]): Try[Int] =
+    Try(hashUnsafe(obj))
```

Comment:
HashWriter is in many cases the same as the JsonWriter. And that's how Json conversion is done right now. In reality, I don't think the writers intentionally throw errors, but it's more like a safety measure.